src/server/index.js

```diff
@@ -117,6 +151,11 @@ stepsResource.delete.fetch(async (req, res, context) => {
   return context.continue;
 });
 
+stepsResource.update.write.after((req, res, context) => {
```

Comment:
what are we doing here? - for my own knowledge

Reply:
Realtime updating the UI. Without refreshing the page